concepts-wip/conditionals/about.md

```diff
@@ -0,0 +1,136 @@
+# Conditionals
+
+Crystal has what is known as Control expressions, these are used to control the way the program will run and they take a truthy or falsey value.
+There are operators that can be used to create truthy or falsey values, these are known as [equality and comparison operators][equality-and-comparison].
+
+There are 2 main control expressions that are used to control which code will run and which will not.
+Also known as which given branch will run.
+
+That is the `if` and the `unless` expression.
+
+## Comparison operators
+
+Comparison operators are used to compare values and return a `true` or `false` value.
+These operators require 2 values to be compared of the same type.
+If the values are not of the same type then the compiler will throw an error.
+Here is a list of the comparison operators and an example of when they give a `true` value:
+
+| Method | Description   | Example |
+| ------ | ------------- | ------- |
+| <      | less          | 5 < 4   |
+| <=     | less or equal | 4 <= 4  |
+| >      | more          | 3 > 1   |
+| >=     | more or equal | 2 >= 2  |
```

Comment:
```suggestion
| <      | less than             | 5 < 4   |
| <=     | less than or equal    | 4 <= 4  |
| >      | greater than          | 3 > 1   |
| >=     | greater than or equal | 2 >= 2  |
```

Reply:
I see from the exemplar that crystal has a "between" operation
```crystal
10 <= x <= 20
```
You could mention it here.